prowler/providers/aws/services/efs/efs_service.py

```diff
@@ -164,7 +164,7 @@ class FileSystem(BaseModel):
     id: str
     arn: str
     region: str
-    policy: Optional[dict] = {}
+    policy: Optional[dict]
```

Comment:
Do you think that we should make this? 
```suggestion
    policy: Optional[dict] = None
```
It's done this way in `prowler/providers/aws/services/s3/s3_service.py`